forms.py

```diff
@@ -1,8 +1,11 @@
 from django import forms
+
 from .models import CATEGORIES
 
+
+# TODO: use forms for rendering!
```

Comment:
Is this TODO done or still pending?

Reply:
This is maybe for future updates, it's not necessary but adds to the quality of code.

Reply:
Then create an Issue in github and remove the todo.

Reply:
I removed a TODO.